editor/editor.go

```diff
@@ -205,7 +207,9 @@ func (e *CodeEditor) load(headerText string) {
 		log.Printf("Error stating file %s: %s", e.filepath, err)
 		return
 	}
+	e.lock.Lock()
 	e.lastModified = finfo.ModTime()
+	e.lock.Unlock()
```

Comment:
I'd like to move this to a different method (e.g. `setLastModified(time.Time)`) so that we can use `defer`.